tests/feature_tests/ci/test.sh

```diff
@@ -11,34 +11,31 @@ export PROXY_PORT=8080
 docker compose build --progress quiet database api_server renderer frontend www
 docker compose up --detach database ldap
 cd components/api_server || exit
-python3 -m venv venv
-. venv/bin/activate
+uv venv
 ci/pip-install.sh
-coverage erase
-RENDERER_HOST=localhost python tests/quality_time_api_server_under_coverage.py &> ../../build/quality_time_api_server.log &
+.venv/bin/coverage erase
+RENDERER_HOST=localhost .venv/bin/python tests/quality_time_api_server_under_coverage.py &> ../../build/quality_time_api_server.log &
 sleep 5  # Give server time to start up
-deactivate
 cd ../..
 # We need to start a second API-server for the renderer. We start it after the API-server under coverage so
 # we can measure the coverage of the startup code, including the containers that depend on the API-server.
 docker compose up --detach api_server renderer frontend www
 cd tests/feature_tests
-python3 -m venv venv
-. venv/bin/activate
+uv venv
 ci/pip-install.sh
 cd ../..
 sleep 5  # Give components time to start up
-coverage erase
-coverage run -m behave --format pretty "${1:-tests/feature_tests/src/features}"
+tests/feature_tests/.venv/bin/coverage erase
```

Comment:
Why do we `cd ../..` above, when everything is then calling the previous path? If it's only for the combine step, perhaps that one can be prefixed?

Reply:
I don't remember exactly but it's probably to make sure the coverage files don't end up in `tests/feature_tests`.